README.md

```diff
@@ -43,22 +43,22 @@ hub credibility.
 
 These and prior disagreements have now made clear the need for an alternative
 hub with a renewed focus and Alignment to serve as the canonical minimal
-IBC/ICS token hub with respect to Cosmos to champion the ideals of sovereignty,
+IBC/ICS token hub for Cosmos to champion the ideals of sovereignty,
 security, and decentralization everywhere; and secondarily to serve as the main
 base for a political party and more-intelligent voting bloc with respect to
-Gaia to save Gaia from itself. A modification to the distribution of $ATONE
-through slashing those who voted in favor of #848 would help ensure that the
+Gaia to save Gaia from itself. A modification to the distribution of $ATONE,
+a through slashing of those who voted in favor of #848 would help ensure that the
```

Comment:
did you mean "through *a* slashing *of*"? or "such as through a slashing of"? 
i still think the previous version makes more sense. 
the slashing isn't necessarily singular,
and I think it's better without the comma.

Reply:
Yes, I meant "through a slashing of".  
We can also use a comma after a shorter phrase when we want to emphasize it or add a pause for literary effect.